### ISSUE TITLE
JAWS not reading form field placeholder text with arrow navigation

### ISSUE THREAD

**Issue by TaylorBlair** [FreedomScientific/VFO-standards-support#688]
### Summary
When using arrow keys to navigate a form, the placeholder text is not read. You can hear JAWS enter forms mode, but the placeholder text is not read. Other screen readers, such as  BVDA, reliably read the placeholder text regardless of navigation method. Users may miss important information.

### Expected result
The placeholder text should be read when moving into the form field.

### Actual result
Using arrow navigation the placeholder text is not initially read. Using other navigation methods such as tab or other applicable hotkeys the placeholder text is read reliably. 

### Example
1. Go to https://www.w3.org/WAI/tutorials/forms/instructions/#placeholder-text
2. Use arrow navigation to enter the form field > the placeholder text will not be read
3. Arrow past the field and then backwards into it > the placeholder text will be read
4. Use tab or other hotkeys to navigate to the field > placeholder text is read

### Additional Information
Possibly related issues:
- [JAWS not narrating the "placeholder" attribute of labelled textarea element #651](https://github.com/FreedomScientific/VFO-standards-support/issues/651)
- [JAWS does not announce the required state on textareas when the placeholder attribute is used #553](https://github.com/FreedomScientific/VFO-standards-support/issues/553)

#### JAWS version and build number
JAWS Version 2022.2211.7 ILM
 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome
Edge

**Comment by JAWS-test** [comment]
I can't reliably reproduce the problem with JAWS 2022 and 2023 + Chrome. It occurs sometimes, but not most of the time, even when I navigate to the field with the virtual cursor and forms mode is automatically activated. It should be said, however, that automatically switching to forms mode when reading with the virtual cursor is and was quite error-prone and should be avoided if possible.
